Bottom: 18f2088923dcedae2201dbd7f20b86718f96d47d
Top:    665d83f8d8f8c943e0ea6ef97c2c3f921baa9609
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-21 23:15:56 +0000

Invalidate outline layer when ....


---

diff --git a/src/polygon.c b/src/polygon.c
index b6a2807..6efb6eb 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1737,6 +1737,10 @@ ClearFromPolygon (DataType * Data, int type, void *ptr1, void *ptr2)
           strcmp (layer->Name, "route") == 0)
         Data->outline_valid = false;
     }
+  else if (type == PIN_TYPE || type == VIA_TYPE)
+    {
+        Data->outline_valid = false;
+    }
 
   if (type == POLYGON_TYPE)
     InitClip (PCB->Data, (LayerType *) ptr1, (PolygonType *) ptr2);
